src/modules/vendor/components/companies/ProfileBilling.vue

```diff
@@ -1,75 +1,80 @@
 <template>
   <q-page class="vendor-background q-pb-xl">
     <div class="q-mb-xl">
-      <p class="text-weight-bold">Suivi des factures</p>
-      <ni-expanding-table :data="courseBills" :columns="columns" v-model:pagination="pagination" :hide-bottom="false"
-        :loading="loading">
-        <template #row="{ props }">
-          <q-td v-for="col in props.cols" :key="col.name" :props="props">
-            <template v-if="col.name === 'number'">
-              <div class="clickable-name" @click.stop="downloadBill(props.row._id)" :disable="pdfLoading">
-                {{ col.value }}
+      <div v-for="payer of Object.keys(courseBills)" :key="payer">
+        <div class="text-weight-bold q-mt-lg q-mb-sm">
+          Formations facturées à {{ courseBills[payer][0].payer.name }}
```

Comment:
En front j'ai l'erreur suivante: `TypeError: undefined is not an object (evaluating '$setup.courseBills[payer][0].payer.name')`